README.md

```diff
@@ -6,26 +6,42 @@ This is the internal UIUC git repository for the DARPA CMAAS inference pipeline.
 <details>
 <summary> Installing </summary>
 
-  To get started with this pipeline you will need to clone the repository and and install [requirements.txt](https://git.ncsa.illinois.edu/criticalmaas/pipeline/-/blob/abode_pipeline/requirements.txt). We recommend using python venv here to keep the working environment clean.
+  To get started with this pipeline you will need to clone the repository. We recommend using python venv here to keep the working environment clean.
 
   ```bash
-  # If you are on hydro you will need to load the python and cuda module.
-  # module load python/3.9.13 cuda/11.7.0 
-
   git clone git@git.ncsa.illinois.edu:criticalmaas/pipeline.git
-  cd pipeline_inference
-  python3 -m venv venv
-  source ./venv/bin/activate
-  pip install -r requirements.txt
+  or git clone https://github.com/DARPA-CRITICALMAAS/uiuc-pipeline.git
+
+  cd uiuc-pipeline
+  ```
+
+  If you are on hydro for the first time, you will need to load the anaconda3_gpu module and create a new conda environment. If conda is already installed on your system, you can skip these two lines.
+
+  ```bash
+  module load anaconda3_gpu
+  conda init
   ```
 
-  This repository also makes use of submodules which will need to be initialized.
+  This repository also makes use of submodules which need to be initialized and updated.
 
   ```bash
   git submodule init
   git submodule update
   ```
 
+  We now create new conda and venv environments and install the [requirements.txt](https://git.ncsa.illinois.edu/criticalmaas/pipeline/-/blob/abode_pipeline/requirements.txt).
+
+  ```bash
+  conda create --name CMAAS_test python=3.10
```

Comment:
Change conda environment name to something more generic. Perhaps `CMAAS_py_3.10`?